airflow/providers/openlineage/extractors/manager.py

```diff
@@ -178,7 +178,7 @@ def extract_inlets_and_outlets(
 
     def get_hook_lineage(self) -> tuple[list[Dataset], list[Dataset]] | None:
         try:
-            from airflow.lineage.hook import get_hook_lineage_collector
+            from airflow.providers.common.compat.lineage.hook import get_hook_lineage_collector
```

Comment:
You need to bump provider dependency for `common.compat` provider for it to work.

Reply:
Yep, the `common.compat` will be released with this PR, which is one of the reasons we can not bump it as it's not yet released. Had a discussion with TP. We'll not use compact in this PR but will bump it after the release.